tests/integration/compose/docker_compose_minio.yml

```diff
@@ -39,6 +39,10 @@ services:
     depends_on:
       - proxy1
       - proxy2
+    volumes:
+      - type: ${RESOLVER_LOGS_FS:-tmpfs}
+        source: ${RESOLVER_LOGS:-}
+        target: /var/log/resolver
```

Comment:
Some tests run scripts like [unstable_proxy.py](https://github.com/ClickHouse/ClickHouse/blob/master/tests/integration/test_merge_tree_s3/s3_mocks/unstable_proxy.py) so it's better to be able to see the results of those tests too. So after this PR we'll see such results in a folder named `_instances/resolver`